apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/WriteTests.cs

```diff
@@ -81,11 +81,11 @@ public async Task WriteValues()
             SpannerNumeric?[] nArray = { SpannerNumeric.Parse("0.0"), null, SpannerNumeric.Parse("2.0") };
             string[] jsonArray = { "{\"f1\":\"v1\"}", "{}", "[]", null };
             string[] sArray = { "abc", null, "123" };
-            string[] bArrayArray =
+            byte[][] bArrayArray =
```

Comment:
I'd suggest adding a second `BytesArray` field, so we can test that both base64-string and real-byte-array representations work. (Basically, I want to keep testing what we're already testing, rather than removing an existing test and replacing it with another.)